runtime/convertValues.go

```diff
@@ -1382,6 +1451,90 @@ func (i valueImporter) importDictionaryValue(
 	), nil
 }
 
+func (i valueImporter) importInclusiveRangeValue(
+	v cadence.InclusiveRange,
+	expectedType sema.Type,
+) (
+	*interpreter.CompositeValue,
+	error,
+) {
+	// start, end and step. The order matters.
+	members := make([]interpreter.IntegerValue, 3)
```

Comment:
Can we just have three variables for `start`, `end` and `step` values (and for types)? creating multiple slices for every range value seems unnecessary.
e.g. even though a slice was created for `memberTypes`, only `memberTypes[0]` is used.

Reply:
Removed `memberTypes` and `memberStaticTypes` slices as they were not needed. Still keeping the `members` slice as it helps with the code reuse while importing members in the below snippet.

```go
	// import members.
	for index, value := range []cadence.Value{v.Start, v.End, v.Step} {
		importedValue, err := i.importValue(value, memberType)
		if err != nil {
			return nil, err
		}
		importedIntegerValue, ok := importedValue.(interpreter.IntegerValue)
		if !ok {
			return nil, errors.NewDefaultUserError(
				"cannot import inclusiverange: start, end and step must be integers",
			)
		}

		members[index] = importedIntegerValue
	}
```